ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/QuerySubscriber.java

```diff
@@ -93,9 +94,19 @@ private void checkMakeRequest() {
 
   @Override
   public void handleError(final Throwable t) {
-    log.error("Error in processing query", t);
-    final KsqlErrorMessage errorResponse = new KsqlErrorMessage(ERROR_CODE_SERVER_ERROR,
-        "Error in processing query. Check server logs for details.");
+    final StringBuilder stringBuilder = new StringBuilder();
+    stringBuilder.append(t);
```

Comment:
Add all the suppressed exceptions to the message exchanged between hosts but also to the one returned to the user